packages/serverpod_client/lib/src/serverpod_client_io.dart

```diff
@@ -17,13 +17,15 @@ import 'serverpod_client_shared_private.dart';
 abstract class ServerpodClient extends ServerpodClientShared {
   late HttpClient _httpClient;
   bool _initialized = false;
+  late Duration _timeout;
```

Comment:
I made the changes you requested. I wonder if this should be non-`final` though to allow the duration to be changed without creating a new client?

I also used `this.timeout = timeout` to be a little more explicit...

Code generation is broken right now -- #1250.

Also note that this PR may conflict with #1012 -- does that one need a unit test, or can it be merged? It's also very straightforward...

Reply:
We're working on fixing the tests and code generation. It will be up very soon!

Reply:
I think `this.timeout = timeout` may cause a linter error.
Making it editable would be nice if it works.

Reply:
> We're working on fixing the tests and code generation. It will be up very soon!

Shouldn't incomplete changesets be committed on a feature branch? Main should always be error-free.

Reply:
> I think `this.timeout = timeout` may cause a linter error. Making it editable would be nice if it works.

There's no linter error in VS Code, at least.